src/main/java/net/minecraftforge/common/brewing/BrewingRecipeRegistry.java

```diff
@@ -19,60 +19,29 @@
 
 package net.minecraftforge.common.brewing;
 
-import java.util.ArrayList;
+import net.minecraft.core.NonNullList;
+import net.minecraft.world.item.ItemStack;
+import net.minecraft.world.item.crafting.RecipeType;
+import net.minecraft.world.level.Level;
+
+import javax.annotation.Nullable;
 import java.util.Collections;
 import java.util.List;
-import net.minecraft.world.item.ItemStack;
-import net.minecraft.world.item.crafting.Ingredient;
-import net.minecraft.core.NonNullList;
 
 public class BrewingRecipeRegistry {
```

Comment:
Does this class need to exist at all? It can just be grabbed through the recipe manager.

Reply:
No because one of my goals is to still be able to use non vanilla brewing stands which is only possible with the canBrew and brewPotions methods

Reply:
Then this wouldn't be registry but a helper, and should be documented as such. This class isn't needed whatsoever since you can recreate the method yourself via the `RecipeManager`, but it's just nice to have around.